test/jit/test_list_dict.py

```diff
@@ -134,6 +134,15 @@ def fn(x):
                 del x[1:3]
                 return x
 
+        with self.assertRaisesRegex(torch.jit.frontend.UnsupportedNodeError,
```

Comment:
Also, I discussed with others and we should move this to `test_builtins.py`

Reply:
Done.